Platform/OpenCanopy/OcBootstrap.c

```diff
@@ -104,6 +104,7 @@ OcShowMenuByOc (
   OcSetInitialCursorOffset ();
   mGuiContext.BootEntry            = NULL;
   mGuiContext.ReadyToBoot          = FALSE;
+  mGuiContext.DoneIntroAnimation   = FALSE;
```

Comment:
This is the fundamental change, it re-runs the menu intro animation every time back into the menu. This is _possibly_ not ideal (i.e. I understand previously it was only supposed to run the first time in), but a) it looks fine, b) it also works, and this is fiddly to fix!

Reply:
No comment in the code :(

Reply:
This?

Reply:
LGTM